styles/tab-background.css

```diff
@@ -29,11 +29,7 @@
 }
 
 .crit-wounds {
```

Comment:
crit wounds are not in the background tab anymore. If you still need this, please move it, otherwise it can be deleted